decayAmplitude/diffractiveDissVertex.cc

```diff
@@ -336,3 +338,10 @@ diffractiveDissVertex::printPointers(ostream& out) const
 	    << "recoil particle = " << recoil()    << endl;
 	return out;
 }
+
+double diffractiveDissVertex::getTprime() const{
+	const TLorentzVector lvBeam = TLorentzVector(beam()->lzVec());
```

Comment:
But the t' function does not have references as argument, so it does not change 4-vectors, isn't it?